securedrop/debian/rules

```diff
@@ -68,11 +68,13 @@ override_dh_installinit:
 override_dh_systemd_enable:
```

Comment:
Unfortunately I wrote "two" in the comment above, can we tweak that to not use a specific number?

"...except those that are run by timers"?